Demo/src/main/java/com/paypal/android/ui/card/CardViewModel.kt

```diff
@@ -36,8 +44,12 @@ class CardViewModel : ViewModel() {
         // Invoke Card SDK
     }
 
-    private fun formatExpirationDate(expirationDate: String): String {
-        return if (expirationDate.length == 2) "$expirationDate/" else expirationDate
+    fun onPrefillCardSelected(cardName: String) {
+        autoFillCards.find { it.first == cardName }?.second?.apply {
```

Comment:
qq: does this pick the second card?

Reply:
`autoFillCards` is a list of pairs, so on this line `.second` would be referencing the `Card` object.